src/views/Peers.tsx

```diff
@@ -99,11 +109,28 @@ export const Peers = () => {
             message.loading({ content: 'Deleting...', key: deleteKey, style });
         } else if (deletedPeer.success) {
             message.success({ content: 'Peer has been successfully removed.', key: deleteKey, duration: 2, style });
+            dispatch(peerActions.resetDeletedPeer(null))
```

Comment:
Codacy has a fix for the issue: [Replace `······dispatch(peerActions.resetDeletedPeer(null))` with `});⏎······dispatch(peerActions.resetDeletedPeer(null));`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
      });
```